scripts/fast-reboot

```diff
@@ -272,13 +273,45 @@ function backup_database()
     fi
 }
 
+function check_mirror_session_acls()
+{
+    ACL_ARP="missing"
```

Comment:
As discussed: 

Please improve the log handling here by adding a new debug line to indicate start of this function call.